packages/web3-eth/src/rpc_method_wrappers.ts

```diff
@@ -0,0 +1,388 @@
+// Disabling because returnTypes must be last param to match 1.x params
+/* eslint-disable default-param-last */
+
+import { EthExecutionAPI, TransactionWithSender } from 'web3-common';
+import { Web3Context } from 'web3-core';
+import {
+	Address,
+	BlockNumberOrTag,
+	convertObjectPropertiesToValidType,
+	convertToValidType,
+	Filter,
+	HexString32Bytes,
+	HexString8Bytes,
+	HexStringBytes,
+	Uint,
+	Uint256,
+	ValidReturnTypes,
+	ValidTypes,
+} from 'web3-utils';
+import { isHexString32Bytes } from 'web3-validator';
+import {
+	convertibleBlockProperties,
+	convertibleFeeHistoryResultProperties,
+	convertibleReceiptInfoProperties,
+	convertibleTransactionInfoProperties,
+} from './convertible_properties';
+
+import * as rpcMethods from './rpc_methods';
+import { BlockFormatted } from './types';
+
+export const getProtocolVersion = async (web3Context: Web3Context<EthExecutionAPI>) =>
+	rpcMethods.getProtocolVersion(web3Context.requestManager);
+
+export const isSyncing = async (web3Context: Web3Context<EthExecutionAPI>) =>
+	rpcMethods.getSyncing(web3Context.requestManager);
+
+export const getCoinbase = async (web3Context: Web3Context<EthExecutionAPI>) =>
+	rpcMethods.getCoinbase(web3Context.requestManager);
+
+export const isMining = async (web3Context: Web3Context<EthExecutionAPI>) =>
+	rpcMethods.getMining(web3Context.requestManager);
```

Comment:
Why we need these wrappers? Why not just pass the `requestManager` where where we are calling these methods?

Reply:
we can remove this additional layer , by passing requestManager to these functions

Reply:
Addressed in [this](https://github.com/ChainSafe/web3.js/pull/4718/commits/380002cbae93d3bc3c6e5ba3efd7b74f20b6f0be) commit